### PR TITLE
#39 Kundur Example 13

### PR DESCRIPTION
**Pull request by MarianneSaugierRTE**

### CODE REVIEW

File: nrt/data/SMIB/KundurEx13/AVRNoPSS.mo [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+  parameter SIunits.PerUnit EfdMax "Maximum excitation voltage";
+
+  // AVR
+  Blocks.Sources.Constant URefPu(k = URef0Pu) annotation(
+    Placement(visible = true, transformation(origin = {-58, 38}, extent = {{-10, -10}, {10, 10}}, rotation = 0)));
+  Blocks.Interfaces.RealInput UStatorPu(start = UStator0Pu) annotation(
+    Placement(visible = true, transformation(origin = {-104, 0}, extent = {{-20, -20}, {20, 20}}, rotation = 0), iconTransformation(origin = {-56, -40}, extent = {{-20, -20}, {20, 20}}, rotation = 0)));
+  Blocks.Continuous.FirstOrder Transducer(T = TR, k = 1, y_start = UStator0Pu) annotation(
+    Placement(visible = true, transformation(origin = {-58, 0}, extent = {{-10, -10}, {10, 10}}, rotation = 0)));
+  Blocks.Math.Add error(k1 = 1, k2 = -1) annotation(
```
[the remark this answers is not in the document]
Comment (MarianneSaugierRTE) [reply nrt/data/SMIB/KundurEx13/AVRNoPSS.mo]: No specific reason... 
Not yet written in the coding style I believe (should we add a convention for individual block naming?)

File: nrt/data/SMIB/KundurEx13/AVRNoPSS.mo [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+  connect(error.y, Exciter.u) annotation(
+    Line(points = {{11, 0}, {24, 0}}, color = {0, 0, 127}));
+  connect(Transducer.y, error.u2) annotation(
+    Line(points = {{-47, 0}, {-31, 0}, {-31, 0}, {-15, 0}, {-15, 0}, {-13, 0}}, color = {0, 0, 127}));
+  connect(UStatorPu, Transducer.u) annotation(
+    Line(points = {{-104, 0}, {-72, 0}, {-72, 0}, {-71, 0}, {-71, 0}, {-70.5, 0}, {-70.5, 0}, {-70, 0}}, color = {0, 0, 127}));
+  connect(efdPuPin.value, efdPu);
+  annotation(
+    uses(Modelica(version = "3.2.2")),
+    Diagram);
```
[the remark this answers is not in the document]
Comment (MarianneSaugierRTE) [reply nrt/data/SMIB/KundurEx13/AVRNoPSS.mo]: Yes it has been automaticaly added by OM

File: nrt/data/SMIB/KundurEx13/AVRNoPSS.mo [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+model AVRNoPSS "Simple Proportional Voltage Regulator"
+  import Modelica.Blocks;
+  import Modelica.SIunits;
+  import Dynawo.Connectors;
+
+  // AVR parameters
+  parameter SIunits.PerUnit KA "Exciter gain";
+  parameter SIunits.Time TR "Transducer time constant";
+  parameter SIunits.PerUnit EfdMin "Minimum excitation voltage";
+  parameter SIunits.PerUnit EfdMax "Maximum excitation voltage";
```
[the remark this answers is not in the document]
Comment (MarianneSaugierRTE) [reply nrt/data/SMIB/KundurEx13/AVRNoPSS.mo]: Yes it is in p.u. I kept the notations of the Kundur book. Should I should the coding style for these models even though they are not integrated in the library?

File: nrt/data/SMIB/KundurEx13/AVRNoPSS.mo [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+
```
[the remark this answers is not in the document]
Comment (MarianneSaugierRTE) [reply nrt/data/SMIB/KundurEx13/AVRNoPSS.mo]: I prefer to use PerUnit to have the right unit if I simulate the model in OpenModelica

File: nrt/data/SMIB/KundurEx13/AVRNoPSS.mo (hunk 2 above)
[the remark this answers is not in the document]
Comment (adriguir) [reply nrt/data/SMIB/KundurEx13/AVRNoPSS.mo]: So we can keep it.

File: nrt/data/SMIB/KundurEx13/AVRNoPSS.mo (hunk 1 above)
[the remark this answers is not in the document]
Comment (adriguir) [reply nrt/data/SMIB/KundurEx13/AVRNoPSS.mo]: Let it this way for now

File: nrt/data/SMIB/KundurEx13/AVRNoPSS.mo (hunk 3 above)
[the remark this answers is not in the document]
Comment (adriguir) [reply nrt/data/SMIB/KundurEx13/AVRNoPSS.mo]: It is better to stick to Kundur book notations so it is ok

**Pull request merged**